GoogleSignIn/Tests/Unit/GIDEMMErrorHandlerTest.m

```diff
@@ -66,6 +66,11 @@ - (void)setUp {
   _isIOS10 = [UIDevice currentDevice].systemVersion.integerValue == 10;
   _keyWindowSet = NO;
   _presentedViewController = nil;
+  UIWindow *fakeKeyWindow = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+  [GULSwizzler swizzleClass:[GIDEMMErrorHandler class]
```

Comment:
Should we test -keyWindow internals rather than mocking?

Reply:
I couldn't find a way to effectively test it (i.e., not just repeat what the code does), because the correctness of this method really depends on the return value of the system APIs -- any suggestions?

Reply:
You could mock connectedScenes/windows/keyWindow and check that window selection happens as expected.  May not be worth it, I leave it to you.

Reply:
Added a test for the `keyWindow` method by mocking its behavior. Not feeling very good about it, but hopefully better than nothing.